report/os_unix.go

```diff
@@ -0,0 +1,32 @@
+// +build darwin linux openbsd freebsd netbsd dragonflypackage report
```

Comment:
```suggestion
// +build linux openbsd freebsd netbsd dragonfly

```
Small typo. Additionally linter suggests that : `+build comment must appear before package clause and be followed by a blank line`

Reply:
Good catch on the issue in the end, but the darwin part was correct. Removed this files entirely and moved it into the `os_xdg` and `os_darwin` files instead.